pronto_mia/lib/ui/components/navigation_menu/navigation_menu.dart

```diff
@@ -49,13 +49,18 @@ class NavigationMenu extends StatelessWidget {
       _buildNavigationCategory(
         'Übersicht',
         [
-          ListTile(
-            leading: const Icon(Icons.today),
-            title: const Text('Einsatzpläne'),
-            onTap: () => model.navigateTo(
-              const DeploymentPlanOverviewView(),
+          if (model.data == null ||

```

Comment:
Why show the menu if `data==null`?

Reply:
I reversed the condition. If data == null the widget is not shown.